docs/scos/dev/feature-integration-guides/202200.0/glue-api/glue-json-api-convention-integration-guide.md

```diff
@@ -0,0 +1,502 @@
+
+$config[GlueJsonApiConventionConstants::GLUE_DOMAIN]
+    = sprintf(
+        'https://%s',
+        $sprykerGlueStorefrontHost ?: 'localhost',
+    );
+```
+
+### 3) Set up transfer objects
+
+Generate transfers:
+
+```bash
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure the following transfers have been created:
+
+| TRANSFER              | TYPE  | EVENT   | PATH                                                            |
+|-----------------------|-------|---------|-----------------------------------------------------------------|
+| GlueApiContext        | class | created | src/Generated/Shared/Transfer/GlueApiContextTransfer.php        |
+| GlueResponse          | class | created | src/Generated/Shared/Transfer/GlueResponseTransfer.php          |
+| GlueRequest           | class | created | src/Generated/Shared/Transfer/GlueRequestTransfer.php           |
+| GlueRequestValidation | class | created | src/Generated/Shared/Transfer/GlueRequestValidationTransfer.php |
+| GlueError             | class | created | src/Generated/Shared/Transfer/GlueErrorTransfer.php             |
+| GlueResource          | class | created | src/Generated/Shared/Transfer/GlueResourceTransfer.php          |
+| GlueRelationship      | class | created | src/Generated/Shared/Transfer/GlueRelationshipTransfer.php      |
+| GlueSparseResource    | class | created | src/Generated/Shared/Transfer/GlueSparseResourceTransfer.php    |
+| GlueFilter            | class | created | src/Generated/Shared/Transfer/GlueFilterTransfer.php            |
+| GlueLink              | class | created | src/Generated/Shared/Transfer/GlueLinkTransfer.php              |
+| Pagination            | class | created | src/Generated/Shared/Transfer/PaginationTransfer.php            |
+| Sort                  | class | created | src/Generated/Shared/Transfer/SortTransfer.php                  |
+
+{% endinfo_block %}
+
+### 4) Set up glue API module resource and route provider plugins
+
+**src/Pyz/Glue/DummyModuleApi/Plugin/DummyResource.php**
+```php
+<?php
+
+namespace Spryker\Glue\DummyModuleApi\Plugin;
+
+use Generated\Shared\Transfer\GlueResourceMethodCollectionTransfer;
+use Generated\Shared\Transfer\GlueResourceMethodConfigurationTransfer;
+use Spryker\Glue\DummyModuleApi\Controller\ResourceController;
+use Spryker\Glue\GlueApplication\Plugin\GlueApplication\AbstractResourcePlugin;
+use Spryker\Glue\GlueJsonApiConventionExtension\Dependency\Plugin\JsonApiResourceInterface;
+
+class DummyResource extends AbstractResourcePlugin implements JsonApiResourceInterface
+{
+    /**
+     * @return string
+     */
+    public function getType(): string
+    {
+        return 'dummyResourceType';
+    }
+
+    /**
+     * @uses \Spryker\Glue\DummyModuleApi\Controller\ResourceController
+     *
+     * @return string
+     */
+    public function getController(): string
+    {
+        return ResourceController::class;
```

Comment:
This will obviously not work for anyone without declaring the class itself. Let's instead use an anonymous class that extends the correct abstraction and has one method only for getCollection (just to make the code shorter).